OpenRA.Mods.Common/Util.cs

```diff
@@ -228,5 +230,22 @@ public static string FriendlyTypeName(Type t)
 
 			return t.Name;
 		}
+
+		public static int GetProjectileInaccuracy(int baseInaccuracy, InaccuracyType inaccuracyType, ProjectileArgs args)
```

Comment:
Also im not a really big fan of using a Utility class where we can just throw stuff into that we have hard to find a home for.

Reply:
That is exactly what I said, but since we have no better place I was told by @pchote to put it here. I was considering making a base projectile class, but that was out of scope.